src/rules/onlyArrowFunctionsRule.ts

```diff
@@ -41,7 +45,9 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class OnlyArrowFunctionsWalker extends Lint.RuleWalker {
     public visitFunctionDeclaration(node: ts.FunctionDeclaration) {
-        this.addFailure(this.createFailure(node.getStart(), "function".length, Rule.FAILURE_STRING));
+        if (this.getOptions().indexOf("allow-declarations") === -1) {
```

Comment:
Could we make a `const OPTION_ALLOW_DECLARATIONS = "allow-declarations";` and use it here and in the two documentation cases above?